xbmc/cores/AudioEngine/Sinks/pipewire/Pipewire.cpp

```diff
@@ -18,12 +18,7 @@
 
 using namespace std::chrono_literals;
 
-namespace AE
-{
-namespace SINK
-{
-namespace PIPEWIRE
-{
+using namespace KODI::PIPEWIRE;
```

Comment:
I would split this into

```c++
using namespace KODI;
using namespace PIPEWIRE;
```

That way you can bring in other KODI namespace members without then needing a `using namespace KODI;` statement.

Reply:
updated